lib/src/license_detection/crc32.dart

```diff
@@ -0,0 +1,279 @@
+/// Specification of [CRC-32][1] is same as used in IEEE 802.3 Ethernet (Normal).
```

Comment:
```suggestion
/// Specification of [CRC-32][1] is same as used in IEEE 802.3 Ethernet (Normal).
///
```

See [effective dart on documentation](https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary): it's generally preferred to start with a single sentence (summary), then give addtional information as paragraphs.

In markdown paragraphs have to be separate by empty lines.